spread.yaml

```diff
@@ -9,6 +9,8 @@ backends:
             - ubuntu-20.04
             - ubuntu-devel:
                 image: ubuntu-daily:devel/amd64
+            - debian-sid:
+                image: debian-cloud-testing/debian-sid
```

Comment:
We use LXD for wlcs CI, which has a different set of images to GCE.

`lxc image list images:debian` lists a bunch of images; we probably want `debian/sid`, and the way the LXD backend of spread works means that the system name `debian-sid` will find the right image without being manually specified.